File: Cpp/inputformat.cpp
#include <iostream> // includes cin to read from stdin and cout to write to stdout
#include <vector>

using namespace std; // since cin and cout are both in namespace std, this saves some text

//There's several format that you can use for C++
int main() {
	/////////////////////////////////////////////////////////////////////////////////////
	//Option #1, this one is pretty simple, found it from GCJ.
    int t, n, m;
    cin >> t; // Input how many case
    
	//If you have multiple case, we can save the outputs in array so we can print it later.
	//most of the time, the output will be produced after all cases inputs
	int output[t];

    //Loop per case.
    for (int i = 0; i < t; ++i) { 
		//Just extra notes we strictlty begin with i=0, not 1, just because we want to put the output into array
		cin >> n >> m; // read n and then m, this is example for 2 integer input.
		cout << "Case #" << i << ": " << (n + m) << " " << (n * m) << endl;
    
		output[t] = 1; //save the results to output.
    }

	//Loop for ouptut, most of the time, the output will be produced after all cases inputs
	for (int i = 0; i < t; i++) {
		cout << output[t] << endl; //Depend on the problems, you might not use endl(new line).
	}

	/////////////////////////////////////////////////////////////////////////////////////

	//Option #2, HackerRank Format, looks a bit unfamiliar at first but I think there's  good reason for it.
	//It usually use multiple function (usually 1 function to process from input to the output).


	/////////////////////////////////////////////////////////////////////////////////////

	//Array Input is used to read line of numbers, this used a lot in Competitive Programming
	//Array Input Option #1


      return 0;
    }

File: python/inputformat.py
t = int(input())
for i in range (1, t+1):
    # WRITE YOUR INPUT SETUP, below will be some of its examples
    x = input() # Single Input, you can assign it to int() function if you want to ensure it is integer
    a, b = [int(s) for s in input().split(" ")] # input 2 integer in a line that separated with space, assign it to a and b
    list = [int(s) for s in input().split(" ")]  # read whatever many integer you write in a line

    # Trivia : Most of the time you either will handle Integer or String input in Competitive Programming

    # WRITE YOUR CODE HERE

    print("Case #{}: {}".format(i, output)) #OUTPUT, you might need to modify it if it produce >1 output